### PR TITLE
chore(ui-tests): add ui visual regression tests

### LINKED ISSUE #6: chore(tests): add ui tests

### PR DESCRIPTION
**Pull request by YonatanKra**
Closes #6

### CODE REVIEW

File: config/rollup/rollup.config.spec.ts [hunk 1]
```diff
@@ -7,7 +7,9 @@ rollupBaseConfig.plugins.push(istanbul({
     exclude: ['src/**/*.spec.ts', 'node_modules/**/*']
 }));
 
-rollupBaseConfig.plugins.splice(1,1, typescript({tsconfig: './config/typescript/tsconfig.spec.json'}),);
+export const specTypescriptConfig = {tsconfig: './config/typescript/tsconfig.spec.json'};
+
+rollupBaseConfig.plugins.splice(1,1, typescript(specTypescriptConfig));
```
Comment (yinonov) [inline config/rollup/rollup.config.spec.ts:12]: what's this really?

File: scripts/visual-tests/index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+
+    </head>
+    <body>
+        <h1>Hello Vivid Tests!</h1>
+    </body>
+</html>
```
Comment (yinonov) [inline scripts/visual-tests/index.html:8]: needed?

File: src/components/badge/ui.test.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
```
Comment (yinonov) [inline src/components/badge/ui.test.ts:1]: is this file redundant?

File: scripts/visual-tests/index.html (hunk 2 above)
Comment (YonatanKra) [reply scripts/visual-tests/index.html:8]: Yes - this is the dev server I'm using to load our tests.

File: src/components/badge/ui.test.ts (hunk 3 above)
Comment (YonatanKra) [reply src/components/badge/ui.test.ts:1]: No - this is the actual test.

File: config/rollup/rollup.config.spec.ts (hunk 1 above)
Comment (YonatanKra) [reply config/rollup/rollup.config.spec.ts:12]: It replaces the prod typescript definition with the spec definition.
I used it elsewhere but eventually didn't need it. but the export remained.

File: src/components/badge/ui.test.ts (hunk 3 above)
Comment (yinonov) [reply src/components/badge/ui.test.ts:1]: I thought we will harvest docs code blocks...?

Comment (YonatanKra) [reply src/components/badge/ui.test.ts:1]: Not at this stage. Not at this stage. This is just the ui-tests mechanism. Do you want it to be part of this PR?

Comment (yinonov) [reply src/components/badge/ui.test.ts:1]: no

**Review by yinonov — APPROVED** [review]

**Pull request merged**